chatbot-conversational_AI/bot-observer-tool/docker/Dockerfile

```diff
@@ -1,15 +1,23 @@
+# Use the official Node.js image with alpine as the base image
 FROM node:alpine
-# Set env 
+
+# Set environment variable
 ENV VERSION 1.0.0
-# Create app directory
-RUN mkdir -p /usr/src/app
+
+# Set the working directory inside the container
 WORKDIR /usr/src/app
+
+# Copy package.json and package-lock.json to the container
+COPY chatbot-conversational_AI/bot-observer-tool/app/package*.json ./
+
 # Install app dependencies
-COPY ./app/package.json ./
-# Install dependencies
 RUN npm install
-# Bundle app source
-COPY ./app ./
+
+# Copy the rest of the application code to the container
+COPY chatbot-conversational_AI/bot-observer-tool/app .
```

Comment:
Would it make sense to do the copy once. I see, in Line 11, package.json files are copied. Then here whole app folder is copied

Reply:
or only copy the server.js. That way we can have a layer with package.json which can be cached.